### DISCUSSION TITLE
Walkthroughs for language tutorials

### DISCUSSION THREAD

**Discussion by worksofliam** [microsoft/vscode-discussions#30] [Extension Development QnA] [answered]
Is anyone using Walkthroughs as code/development tutorials for a new language, or something of the such?

I'm thinking about how I could integrate a language tutorial right into a walkthrough.

**Comment by miguelsolorio** [comment, marked as answer]
It's not too common, but I have seen some instances of this like in the [LTeX extension](https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex) where they open examples:

<img width="1060" alt="CleanShot 2022-06-09 at 09 13 27@2x" src="https://user-images.githubusercontent.com/35271042/172894730-1a76f36c-2508-46e6-940d-78902b84d4b6.png">

<img width="1060" alt="CleanShot 2022-06-09 at 09 15 43@2x" src="https://user-images.githubusercontent.com/35271042/172895262-c4d40eb0-23fa-4a50-a584-705af790debd.png">
